Upstream: no
Reason: since botan 2.17 update some header files reanamed

--- lib/crypto/cipher.cpp
+++ lib/crypto/cipher.cpp
@@ -26,7 +26,7 @@
 #include <stdexcept>
 
 #include <botan/auto_rng.h>
-#include <botan/key_filt.h>
+#include <botan/filters.h>
 #include <botan/lookup.h>
 #include <botan/md5.h>
 #include <botan/pipe.h>
